stone/api.py

```diff
@@ -18,19 +19,21 @@
 
 from stone.data_type import (  # noqa: F401 # pylint: disable=unused-import
     Alias,
+    DataType,
     Struct,
-    UserDefined,
 )
 
 from stone.lang.parser import StoneRouteDef  # noqa: F401 # pylint: disable=unused-import
 
-# TODO: This can be changed back to a single declaration with a unicode
-# literal after <https://github.com/python/mypy/pull/2516> makes it into a
-# PyPi release
-if six.PY3:
-    NamespaceDict = typing.Dict[str, 'ApiNamespace']
-else:
-    NamespaceDict = typing.Dict[str, b'ApiNamespace']
+__MYPY = False
+if __MYPY:
+    # TODO: This can be changed back to a single declaration with a
+    # unicode literal after <https://github.com/python/mypy/pull/2516>
+    # makes it into a PyPi release
+    if six.PY3:
+        NamespaceDict = typing.Dict[str, 'ApiNamespace']
+    else:
+        NamespaceDict = typing.Dict[str, b'ApiNamespace']
```

Comment:
This is a common hack to allow running of Stone code even if the `typing` module is sub-par. It should be possible to extend this such that one could remove the dependency on `typing` altogether, but that's probably the subject of another PR.